apps/odyssey-stats/src/components/odyssey-query-site-purchases.ts

```diff
@@ -12,14 +12,18 @@ import {
 	PURCHASES_SITE_FETCH_COMPLETED,
 	PURCHASES_SITE_FETCH_FAILED,
 } from 'calypso/state/action-types';
+import { getApiNamespace, getApiPath } from '../lib/get-api';
 
 async function queryOdysseyQuerySitePurchases( siteId: number | null ) {
 	if ( ! siteId ) {
 		return;
 	}
 
 	return wpcom.req
-		.get( { path: '/site/purchases', apiNamespace: 'jetpack/v4' } )
+		.get( {
+			path: getApiPath( '/site/purchases', { siteId } ),
```

Comment:
@candy02058912 quick question: With this change the `siteId` will always be passed, which means that the API request for Jetpack sites will now be

`{ path: '/sites/:siteId/purchases', apiNamespace: 'jetpack/v4' }` instead of
`{ path: '/site/purchases', apiNamespace: 'jetpack/v4' }`

Is this expected?

Reply:
No, let me fix that